src/distributed_ls/pilut/DistributedMatrixPilutSolver.h

```diff
@@ -78,31 +78,31 @@ HYPRE_Int Ul_timer;
 
 /* DEFINES for global variables */
 #define pilut_comm (globals->hypre_MPI_communicator)
-#define mype (globals->mype)
```

Comment:
@rfalgout My guess is GCC compilers (new versions, see https://github.com/hypre-space/hypre/issues/126#issue-625738732) got confused from these lines (starting from 81). Must have something to do with name conflicts in, e.g., `#define mype (global->mype)` and `mype` in the struct. Seems working with the new changes.